src/meta/meta.hpp

```diff
@@ -1860,9 +1860,9 @@ class type final {
         std::array<any, sizeof...(Args)> arguments{{std::forward<Args>(args)...}};
         any any{};
 
-        internal::iterate<&internal::type_node::ctor>([data = arguments.data(), &any](auto *node) -> bool {
```

Comment:
Uhm... the `-> bool` here makes me think that this is something I've never update from an older version of the library. `iterate` doesn't require a return value after all for the lambda. It seems odd at least.